server/app/Http/Controllers/UserController.php

```diff
@@ -636,7 +633,7 @@ public function oauthLogin($user_info, $redirect = false)
         }
     }
 
-    public function getSupport($user, $doc)
+    public function getSupport(DocAccessReadRequest $request, $user, Doc $doc)
     {
         $docMeta = DocMeta::where('user_id', $user->id)->where('meta_key', '=', 'support')->where('doc_id', '=', $doc)->first();
```

Comment:
In the `doc_id` where clause it should use `$doc->id` instead of just `$doc` now.